### PR TITLE
Jest routes

### PR DESCRIPTION
**Pull request by this-Cunningham**
tests for all routes so far

### CODE REVIEW

**Comment by this-Cunningham** [comment]
[JEST - server complete](https://trello.com/c/yen4Aczp/63-jest-server-complete)

**Review by sabateklu — APPROVED** [review]
good choice on disabling lint given the inherent field name

**Review by sabateklu — APPROVED** [review]
Server routes and tests look great. very clean.

**Review by sabateklu — APPROVED** [review]
comprehensive testing of the response format. good job!

File: spec/server.test.js [hunk 1]
```diff
@@ -16,17 +16,37 @@ app.use(express.urlencoded({ extended: false }));
 app.use('/', showcase);
 
 describe('showcase Routes', () => {
-  afterEach(() => {
+  afterAll(() => {
```
Comment (sabateklu) [inline spec/server.test.js]: makes sense to keep the connection open here.

**Review by sabateklu — APPROVED** [review]
great job testing for errors as well.

**Review by sabateklu — APPROVED** [review]
testing your patch by the boolean value was a great idea and to also have that built into your router.

**Review by sabateklu — APPROVED** [review]
good thinking adding the tests for the status code

**Review by sabateklu — APPROVED** [review]
clarity in your testing and more concise testing code.

**Review by sabateklu — APPROVED** [review]
great use of err first cb to ensure full coverage.

**Review by sabateklu — APPROVED** [review]

**Review by sabateklu — APPROVED** [review]

**Review by sabateklu — APPROVED** [review]
precision via use of .end()=> .. much cleaner code

**Review by sabateklu — APPROVED** [review]

**Review by sabateklu — APPROVED** [review]

**Review by sabateklu — APPROVED** [review]

**Review by sabateklu — APPROVED** [review]

File: spec/server.test.js [hunk 2]
```diff
@@ -10,10 +10,15 @@ const showcase = require('../server/showcase');
 
 const app = express();
 
-app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
+beforeAll(async () => {
+  app.use(express.json());
+  app.use(express.urlencoded({ extended: false }));
+  app.use('/', showcase);
```
Comment (sabateklu) [inline spec/server.test.js:16]: good idea adding these into the beforeAll block-- efficient!

**Review by sabateklu — APPROVED** [review]

**Pull request merged**